### PR TITLE
ns5x, nv4x: update building instructions for v1.3.0

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: docs/variants/novacustom_ns5x_7x/building-manual.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The easiest way to build coreboot is to use the official Docker image.
 1. Checkout to the desired Dasharo revision:
 
     > Replace the REVISION with one of the:
-    > - `novacustom_ns5x/release` for the latest released version
+    > - `clevo/release` for the latest released version
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x_7x/building-manual.md:34]: @mkopec Have you actually tried to follow thos documentation? I think that the real problem is that there is no `build.sh` script on the `clevo/release` branch.
Maybe we should fall back to using a few commands, as we have for Optiplex, for example: https://github.com/Dasharo/docs/blob/master/docs/variants/dell_optiplex/building-manual.md
Unless there are better suggestions.

**Review by macpijan — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_ns5x_7x/building-manual.md (hunk 1 above)
Comment (macpijan) [reply docs/variants/novacustom_ns5x_7x/building-manual.md:34]: @mkopec ?

Comment (mkopec) [reply docs/variants/novacustom_ns5x_7x/building-manual.md:35]: Fixed

File: docs/variants/novacustom_nv4x/building.md [hunk 2]
```diff
@@ -15,40 +15,52 @@ This documents describes the procedure for compiling coreboot for NovaCustom NV4
...
+    docker run --rm -it -u $UID \
+       -v $PWD:/home/coreboot/coreboot \
+       -w /home/coreboot/coreboot \
+       coreboot/coreboot-sdk:0ad5fbd48d /bin/bash
+    ```
+
+1. Build the firmware:
+
+    ```bash
+    cp configs/config.novacustom_ns5x .config
```
[the remark this answers is not in the document]
Comment (macpijan) [reply docs/variants/novacustom_nv4x/building.md:57]: @PrzemyslawBanasiak13 I assume nv4x should be used here.

File: docs/variants/novacustom_nv4x/building.md [hunk 3]
```diff
@@ -15,40 +15,52 @@ This documents describes the procedure for compiling coreboot for NovaCustom NV4
...
+    make olddefconfig
+    make
+    ```
 
 The resulting coreboot image will be placed in
```
Comment (PrzemyslawBanasiak13) [inline docs/variants/novacustom_nv4x/building.md:62]: I've tested the procedure and after `make` finishes there is no `coreboot.rom` in `build\`.

[Link to my task](https://jira-3mdeb.atlassian.net/browse/NCM-529)

Comment (mkopec) [reply docs/variants/novacustom_nv4x/building.md:60]: Pushed a fix, please retest. Now it built successfully for me

File: docs/variants/novacustom_nv4x/building.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mkopec) [reply docs/variants/novacustom_nv4x/building.md]: Fixed

File: docs/variants/novacustom_nv4x/building.md (hunk 3 above)
Comment (macpijan) [reply docs/variants/novacustom_nv4x/building.md:60]: @PrzemyslawBanasiak13 @sulewskiprzemyslaw Have we retested that?
We need all of the documentation changes live before publishing NV4X release

Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_nv4x/building.md:60]: @macpijan @mkopec I've just tested the whole procedure and it works fine

File: docs/variants/novacustom_nv4x/building.md [hunk 4]
```diff
@@ -11,46 +11,79 @@ This documents describes the procedure for compiling coreboot for NovaCustom NV4
...
+    ```
 
-Build the firmware:
+1. Start the coreboot-sdk Docker container:
 
-```bash
-./build.sh build
-```
+    ```bash
+    docker run --rm -it -u $UID \
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_nv4x/building.md:46]: @mkopec could You refactor this section? Command after copying does not work at all

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_nv4x/building.md (hunk 4 above)
Comment (mkopec) [reply docs/variants/novacustom_nv4x/building.md:46]: Is there an error? It drops into bash inside the container on my machine

Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_nv4x/building.md:46]: @mkopec sorry, I have some internal error, the command works fine

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Review by macpijan — APPROVED** [review]

**Pull request merged**